packages/@aws-cdk-testing/framework-integ/test/aws-rds/test/integ.cluster-instance-id.ts

```diff
@@ -14,9 +14,9 @@ const stack = new cdk.Stack(app);
 const vpc = new ec2.Vpc(stack, 'VPC');
 
 new rds.DatabaseCluster(stack, 'Database', {
-  engine: rds.DatabaseClusterEngine.auroraMysql({ version: rds.AuroraMysqlEngineVersion.VER_3_01_0 }),
+  engine: rds.DatabaseClusterEngine.auroraMysql({ version: rds.AuroraMysqlEngineVersion.VER_3_06_0 }),
   writer: rds.ClusterInstance.provisioned('Instance', {
-    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE3, ec2.InstanceSize.SMALL),
+    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE3, ec2.InstanceSize.MEDIUM),
```

Comment:
why the size increase?

Reply:
8.0.mysql_aurora.3.06.0 can't be configured with an instance size small. It'll cause a deployment failure. The instance size has to be medium or large.